File: Array/Max Min in Array/max_min_array.cpp
#include<iostream>
using namespace std;
main(){
    int a[10],i,max,min;
    cout<<"Enter 10 numbers : ";
    for(i=0;i<10;i++)
        cin>>a[i];
    min=a[0];
    max=a[0];
    for(i=0;i<10;i++){
        if(a[i]>max)
            max=a[i];
        if(a[i]<min)
            min=a[i];
    }
    cout<<"Maximum in Array : "<<max<<endl;
    cout<<"Minimum in Array : "<<min;
}

File: Problem Statements/Array Problems/Max Min in Array/max_min_array.java
import java.util.*;
class max_min_array{
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter 10 numbers : ");
        int a[]=new int[10];
        for(int i=0;i<10;i++)
            a[i]=sc.nextInt();
        sc.close();
        int max=a[0];
        int min=a[0];
        for(int j=0;j<10;j++){
            if(a[j]>max)
                max=a[j];
            if(a[j]<min)
                min=a[j];
        }
        System.out.println("Maximum in Array : "+max);
        System.out.print("Minimum in Array : "+min);
    }
}